tapalcatl_server/handler.go

```diff
@@ -116,6 +121,14 @@ func logBool(x bool) string {
 	}
 }
 
+func convertNanosToMillis(nanoseconds int64) int64 {
+	return nanoseconds / 1000000
```

Comment:
Good call! I made a separate pr for it because I already merged this https://github.com/tilezen/tapalcatl/pull/15